scenario14/jmeter/01-Scenario14-EnforcePasswordReset.jmx

```diff
@@ -1810,7 +1810,7 @@
             <connectTime>true</connectTime>
           </value>
         </objProp>
-        <stringProp name="filename"></stringProp>
+        <stringProp name="filename">/Users/pubudup/Documents/Github/cr/identity-test-integration/scenario14/jmeter/results1.jtl</stringProp>
```

Comment:
Can't we add a relative path here so that we don't need to change the file path in different environments?

Reply:
Actually this is a .jtl result reference which should be removed from the script. We will not need this.